### ISSUE TITLE
Secret of Mana (637670)

### ISSUE THREAD

**Issue by cain05** [ValveSoftware/Proton#5700]
# Compatibility Report
- Secret of Mana
- 637670

## System Information
- GPU: AMD 5700XT
- Driver/LLVM version: Mesa 21.3.7-2
- Kernel version: 5.16.15 Zen
- Proton version: 7.0-1
https://gist.github.com/cain05/acbaf9adafeb4270e99763de0dae1920

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-637670.log](https://github.com/ValveSoftware/Proton/files/8309471/steam-637670.log)

## Symptoms
Opening video does not display anything but the subtitles.  Narration and audio seem to work fine.

## Reproduction
Start a new game, select a character name then the video will play.

**Comment by nsivov** [comment]
This is about EVR component:

`1073.832:013c:0140:fixme:evr:video_presenter_control_GetCurrentImage 12EE003C, 0156FD3C, 0156FCBC, 0156FCA4, 0156FC9C.`

The fix was pushed to experimental, please try to switch the game to "Proton Experimental" for retesting.

**Comment by cain05** [comment]
Finally got around to testing it.  It seems to be working on Proton Experimental as you noted.

**Comment by kisak-valve** [comment]
Secret of Mana (637670)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5909.
@Nagakuro posted on 2022-06-14T23:01:29:

# Compatibility Report
- Name of the game with compatibility issues: Secret of Mana
- Steam AppID of the game: 637670

## System Information
- GPU: AMD Custom APU 0405/Steam Deck
- Driver/LLVM version: Mesa 22.0.0
- Kernel version: 5.13.0-valve15-1-neptune-02197-gf6ec7ad3762a
- Proton version: both 7.0-3 and Proton Experimental as well as Proton 6.3-8 and all Proton GE 7 versions 7.1 to 7.20

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-637670.log](https://github.com/ValveSoftware/Proton/files/8904326/steam-637670.log)


## Symptoms: 
As of the latest Proton Experimental Update on June 14th 2022 the cutscenes in Secret of Mana appear black again now and skipping the cutscenes cause a crash, audio is fully functional, and subtitles display, but before this update literally just yesterday I was able to play it just fine on Proton Experimental, cutscenes were working fully and now they aren't. I tried seeing if the new implementations in 7.0-3 would work and nope, so now I have no way to enjoy the cutscenes in this game anymore. 

And on top of this, no other proton version has a working version with the cutscenes working anymore, sadly. I tried every Proton GE version, as well as all past versions of proton available for download Directly thru steam.


## Reproduction
Video issue:

1. Start a new game, pick language, pick text language, character name, press ok
2. Then the cutscene should start playing with a black screen, but will have audio and subtitles

Freeze:

1. proceeding to push the Start/Skip button for the cutscene, on the steam deck soft locks the game (game froze, and have to force exit) when trying to skip the cutscene.

![steamuserimages-a akamaihd](https://user-images.githubusercontent.com/107516271/173702708-bddda4d3-459a-4391-8d10-c902642e7d1f.jpg)



***
@Nagakuro commented on 2022-06-14T23:14:01:

After further investigation, this issue was brought up 2 months ago, and the response given was 

"This is about EVR component:

1073.832:013c:0140:fixme:evr:video_presenter_control_GetCurrentImage 12EE003C, 0156FD3C, 0156FCBC, 0156FCA4, 0156FC9C.

The fix was pushed to experimental, please try to switch the game to "Proton Experimental" for retesting."

So it seems a regression in the update on June 14th 2022 of Proton Experimental and Proton 7.0-3 was made.

**Comment by Nagakuro** [comment]
So yeah as I was saying in the other thread, seems a regression was made in the latest proton 7.0-3 and proton experimental update. Is there anyway to get the fix pushed back into experimental or no? Cause the game was working practically flawlessly the past couple weeks I've been playing it.

**Comment by Nagakuro** [comment]
So any update on whether this issue was looked into or not? Cause it still seems to be doing it

**Comment by alasky17** [comment]
@Nagakuro I was unable to reproduce the success you reported with an older experimental build.  I've uploaded a build from within the timeframe where you reported success here: https://www.codeweavers.com/xfer/alasky/proton-test/ with lock code "proton".  You can test this build by:

1) create a folder called compatibilitytools.d in this location: ~/.steam/steam/compatibilitytools.d
2) Untar this build inside this folder.  It should live as folder inside the folder.
3) Reboot Steam.  The build should show up in the drop-down menu where you select Proton versions.

Please let me know if this build works for you, or if you think that a build from another date would work (there is also a 0610 build, but I felt that was too close to the date you reported the breakage).

**Comment by Nagakuro** [comment]
> @Nagakuro I was unable to reproduce the success you reported with an older experimental build. I've uploaded a build from within the timeframe where you reported success here: https://www.codeweavers.com/xfer/alasky/proton-test/ with lock code "proton". You can test this build by:
> 
> 1. create a folder called compatibilitytools.d in this location: ~/.steam/steam/compatibilitytools.d
> 2. Untar this build inside this folder.  It should live as folder inside the folder.
> 3. Reboot Steam.  The build should show up in the drop-down menu where you select Proton versions.
> 
> Please let me know if this build works for you, or if you think that a build from another date would work (there is also a 0610 build, but I felt that was too close to the date you reported the breakage).

Sadly that build did not work, is there any builds closer to the original time it got fixed like around April 9th-20th when it was originally reported fixed? I can't remember the exact date it stopped working, but I know it stopped working in June.

**Comment by alasky17** [comment]
@Nagakuro Yep - I have one from 0427.  I believe this should fall squarely in your timeline?  I just uploaded it to the same folder :)

**Comment by Nagakuro** [comment]
> @Nagakuro Yep - I have one from 0427. I believe this should fall squarely in your timeline? I just uploaded it to the same folder :)

Seems now even I am unable to reproduce a success rate on my Steam Deck with either build, which is a bummer, but is what it is. I'm wondering if there may of been a driver update or kernel update on the Steam Deck itself that is now causing it to not function where as before it could. I'll give up for now and keep testing as newer versions of proton come out.

**Comment by alasky17** [comment]
@Nagakuro Thank you for verifying that.  Bummer indeed :/  We are constantly working on improving video playback, and there is a bug filed for this specific failure.  Unfortunately I don't have a timeline on when this specific game will be fixed - it is a hard problem to solve.  I have had some inconsistency when testing this game before, so it seems likely to me that you were getting some very lucky timing and bypassing the bug, and that timing changed (rather than a regression).

**Comment by Nagakuro** [comment]
> @Nagakuro Thank you for verifying that. Bummer indeed :/ We are constantly working on improving video playback, and there is a bug filed for this specific failure. Unfortunately I don't have a timeline on when this specific game will be fixed - it is a hard problem to solve. I have had some inconsistency when testing this game before, so it seems likely to me that you were getting some very lucky timing and bypassing the bug, and that timing changed (rather than a regression).

So Proton Experimental with Bleeding Edge branch had an update today for me on my Steam Deck, and the cutscenes seem to be working now again, is there any way I can get a back up of the current proton experimental bleeding edge build that is out now? Just so for personal use?

EDIT: Nvm I was able to just browse to the local files of Proton Experimental, make a backup in the proton folder where I have all my Proton-GE builds and called it Proton-SOM and made a compatability.vdf file so it would show up, and the custom build is working with cutscenes just fine as well, I'll be sure to post here if at any point cutscenes stop working again, as at that point, it wouldn't be a proton issue, it would be something driver related.

**Comment by alasky17** [comment]
@Nagakuro Awesome!  I am hoping that the most recent experimental release (0706) will also fix Secret of Mana for you - hopefully, non-bleeding-edge is also fixed.  I assume you are seeing the cutscenes upside down right now?

**Comment by Nagakuro** [comment]
> @Nagakuro Awesome! I am hoping that the most recent experimental release (0706) will also fix Secret of Mana for you - hopefully, non-bleeding-edge is also fixed. I assume you are.

Yeah cutscenes are upside down, I didn't even notice that xD RIP, now I guess I gotta wait even more. Dang it, you made me notice it XD I wouldn't of noticed otherwise. Also yes, proton-experimental without bleeding edge also works now as well
